rpc/types_transaction.go

```diff
@@ -93,17 +93,34 @@ func (tx L1HandlerTxn) Hash() *felt.Felt {
 	return tx.TransactionHash
 }
 
-type DeclareTxn struct {
+type DeclareTxnV1 struct {
 	CommonTransaction
 
 	// ClassHash the hash of the declared class
 	ClassHash *felt.Felt `json:"class_hash"`
 
+	DeprecatedContractClass `json:"contract_class"`
```

Comment:
This property is not included in version 03, is it from 04? Should we add it here anyway?
I don't think it'll cause any problems